src/Draft/PlayerList/CurrentBid.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react'
+
+
+class CurrentBid extends Component {
+  constructor(props) {
+    super(props)
+    this.state = { bid: 0, player: {}}
+  }
+
+  componentDidMount() {
+    this.subscription = new Subscription('/draftBid',
```

Comment:
Is /draftBid implemented somewhere or am I not understanding how observables work?

Reply:
Its in streamLib .  Its a wrapper around a get stream so that we can move from:
1. http get with an keep-alive loop (what we have now)

to

2. SSE (what we're gonna swap to because keep-alive doesn't work)

3. duplex http2 streams(we can't do this yet because browsers don't like it and they might never like it but maybe they will)

Reply:
We'll probably move to observables so that we can make joint subscriptions <sup>1</sup> more easily.

1 subscriptions that're opened and closed based off of another subscription.